### PR TITLE
Dj apvideo

### PR DESCRIPTION
**Pull request by djangobotpython**

### CODE REVIEW

File: videos/migrations/0014_apvideo.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (Abhishek-lodha) [inline videos/migrations/0014_apvideo.py]: merge both the migration files

File: videos/models.py [hunk 2]
```diff
@@ -247,3 +248,78 @@ class Meta:
 
     def __unicode__(self):
         return self.vc
+
+
+# class BluefrogSubcategory(models.Model):
```
Comment (Abhishek-lodha) [inline videos/models.py]: remove commented code

File: activities/management/commands/bluefrog_practice_new.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+from geographies.models import *
+#app imports
+from videos.models import *
+import ap_data_integration as ap
+
+
+class Command(BaseCommand):
+	def handle(self, *args, **options):
+		#read xml from url
+		req = requests.get('http://45.127.101.204/DG_API/AP_MIS.svc/GetSoilFertilityManagementDetails', auth=('Bluefrog', 'Blue@123'))
```
Comment (Abhishek-lodha) [inline activities/management/commands/bluefrog_practice_new.py]: store credentials in settings file

File: videos/models.py (hunk 2 above)
Comment (djangobotpython) [reply videos/models.py]: Done

File: activities/management/commands/bluefrog_screening.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+			mediator_code = data_iterable.get('CRP/CA ID')
+			videos = data_iterable.get('Video IDs')
+			no_of_male = data_iterable.get('No. of Male')
+
+			no_of_female = data_iterable.get('No. of Female')
+			total_members = data_iterable.get('Total')
+			screening_duration = data_iterable.get('Duration of Screening')
+
+			# for saving in screening
+			start_date = start_date
```
Comment (Abhishek-lodha) [inline activities/management/commands/bluefrog_screening.py]: redundant

File: activities/management/commands/ap_data_integration.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        body = "Dear Team,\n\n" + "AP data has been successfully updated in COCO.\n\n" + "Here are the details of this update:\n\n" + "Total entries sent by AP (A): " + str(new_count+duplicate_count+other_error_count) + "\nEntries with errors (B): " + str(other_error_count) + "  *e.g. missing video category, missing video in screening, etc. error files are attached.\nEntries we already had (C): " + str(duplicate_count) + "\nEntries successfully updated in COCO (A-(B+C)): " + str(new_count) + "\n\nPlease contact system@digitalgreen.org for any clarification.\n\nThank you.\n"
+        msg = EmailMultiAlternatives(subject, body, from_email, to_email)
+        for files in glob.glob("ap/*"):
+            if files.endswith('.csv'):
+                msg.attach_file(files, 'text/csv' )
+        msg.send()
+
+    def handle(self, *args, **options): 
+        
+        print "Geography,ap_geo"
```
Comment (Abhishek-lodha) [inline activities/management/commands/ap_data_integration.py]: remove print statements

**Pull request merged**